generators/app/templates/javascript/userResolver.test.js

```diff
@@ -93,3 +94,19 @@ test('test login resolver', async () => {
   });
   expect(result.errors[0].message).toBe("No user with that email");
 });
+
+test('test loggedInUser resolver', async () => {
+  const query = `
+    query TEST {
+      loggedInUser {
+        id
+      }
+    }
+  `;
+
+  let result = await tester.graphql(query, undefined, {});
+  expect(result.errors[0].message).toBe("You are not authenticated!");
+
+  result = await tester.graphql(query, undefined, { user: { id: mongoose.Types.ObjectId() } });
+  expect(result.errors).toBe(undefined);
```

Comment:
Is there any way you can test for a logged in user if the user should be logged in?

The easyGraphQL tester package provides a way to pass in context (I believe as the 3rd argument here), which you may be able to set as a fake logged in user.

If you cant get this though, dont worry as I'll add it in at a future date

Reply:
Thanks for the pointer @CrimsonNynja ! Fixed via [6826de8](https://github.com/CrimsonNynja/generator-node-graphql/pull/54/commits/6826de8a5520256a56d4291171d9971492a0d13c) 🤝